docs/backend-system/core-services/root-http-router.md

```diff
@@ -42,6 +42,21 @@ createBackendPlugin({
 
 ## Configuring the service
 
+### Via `app-config.yaml`
+
+The `app-config.yaml` file provides configurable options that can be adjusted to meet your `RootHttpRouterService` specific requirements:
+
+```yaml
+backend:
+  lifecycle:
+    # (Optional) The maximum time that paused requests will wait for the service to start, before returning an error (defaults to 5 seconds).
+    startupRequestPauseTimeout: { seconds: 10 }
+    # (Optional) The maximum time that the server will wait for stop accepting traffic, before returning an error (defaults to 30 seconds).
+    shutdownRequestPauseTimeout: { seconds: 20 }
```

Comment:
The naming here is a bit too symmetrical imo, because these two don't function in the same way.

I'm thinking simply `shutdownRequestPauseTimeout` -> `shutdownDelay` or something like that? It's not really a timeout.

Regarding both of these I'm a bit torn on `backend.lifecycle.*`, some other options would be `backend.*` or `backend.http.*`

Reply:
So as discussed let's stick to `backend.lifecycle.*`

Reply:
@camilaibs this still needs a fix ^

Reply:
Fixed 🙏🏻